app/App.tsx

```diff
@@ -86,6 +85,7 @@ export const App = () => {
 
   return (
     <div id={outerContainerId}>
+      { /* eslint-disable-next-line react/jsx-no-constructed-context-values */ }
```

Comment:
Since `userLocation` comes from an async call, the linter was barking about this. I couldn't come up with a simple fix, so I disabled this line as something more complex seemed out of scope and maybe overkill for appeasing the linter.